Sources/Filters/General/ClipClosedSurface/ccsEdgeLocator.js

```diff
@@ -0,0 +1,30 @@
+export default class CCSEdgeLocator {
+  constructor() {
+    this._edgeMap = new Map();
+  }
+
+  initialize() {
+    this._edgeMap.clear();
+  }
+
+  insertUniqueEdge(i0, i1, i) {
```

Comment:
you do not need to pass `i`, the third parameter is only a returned value in C++